admin/class-config.php

```diff
@@ -135,6 +135,26 @@ public function localize_admin_script() {
 		);
 	}
 
+	/**
+	 * Runs the yoast exporter class to possibly init the file download.
+	 */
+	protected function do_yoast_export() {
+		$can_manage_options = current_user_can( 'manage_options' );
+		$is_nonce_invalid   = ! wp_verify_nonce( filter_input( INPUT_POST, 'export_nonce' ), 'wpseo-export' );
```

Comment:
We can use `check_admin_referrer` here. But make sure you first check if the nonce key exists in the $_POST.